tests/hardhat/Tokens/transferTest.ts

```diff
@@ -46,19 +46,12 @@ describe("VToken", function () {
       await expect(vToken.transfer(rootAddress, 50)).to.be.revertedWithCustomError(vToken, "TransferNotAllowed");
     });
 
-    it("rejects transfer when not allowed and reverts if not verified", async () => {
+    it("rejects transfer when not allowed", async () => {
       await vToken.harnessSetBalance(rootAddress, 100);
       expect(await vToken.balanceOf(rootAddress)).to.equal(100);
 
-      comptroller.transferAllowed.returns(11);
-      await expect(vToken.transfer(rootAddress, 50))
-        .to.be.revertedWithCustomError(vToken, "TransferComptrollerRejection")
-        .withArgs(11);
-
-      //comptroller.transferAllowed.returns(Error.NO_ERROR);
-      //await send(vToken.comptroller, 'setTransferVerify', [false])
-      // no longer support verifyTransfer on vToken end
-      // await expect(send(vToken, 'transfer', [guyAddress, 50])).rejects.toRevert("revert transferVerify rejected transfer");
+      comptroller.preTransferHook.reverts();
+      await expect(vToken.transfer(rootAddress, 50)).to.be.reverted;
```

Comment:
why not keep the check for the correct error?

Reply:
Because there's no single error anymore, it can revert with whatever Comptroller reverts with.

Or do you mean sth like this?
```
comptroller.preTransferHook.reverts("Comptroller had a bad day today");
await expect(vToken.transfer(rootAddress, 50)).to.be.revertedWith("Comptroller had a bad day today");
```

When I tried it last time, smock didn't preserve the error reason, idk why